### ISSUE TITLE
breathe on windows fails due to missing path to doxygen

### ISSUE THREAD

**Issue by psifertex** [michaeljones/breathe#271]
Breathe/sphinx/doxygen on Windows (at least under a Jenkins build environment) fails to run because https://github.com/michaeljones/breathe/blob/master/breathe/process.py#L87 does not find doxygen in the path without an additional`shell=True` option to the subprocess call. The change that worked for me was: 

```
        self.run_process(['doxygen', cfgfile], cwd=build_dir, shell=True)
```

Note that shell=True is considered a potential security risk, however in this case all arguments to the command-line are from local data and so it shouldn't be a concern.

**Comment by vitaut** [comment]
Thanks for catching this. Could you submit a PR?

**Comment by psifertex** [comment]
Certainly -- just did so, sorry, I missed this reply originally.

**Comment by michaeljones** [comment]
Should be fixed in with #277
